pages.de/linux/sysctl.md

```diff
@@ -0,0 +1,23 @@
+# sysctl
+
+> Kernelparameter zur Laufzeit auflisten und ändern. Als Parameter stehen die Werte unter `/proc/sys/` zur Verfügung (in Unterordner organisiert).
```

Comment:
This does seem useful, but we sholudn't add anything which isn't in the English page. But you can of course open a PR to add it there.
```suggestion
> Laufzeit-Kernelparameter auflisten und ändern.
```